JS1/css/style.css

```diff
@@ -0,0 +1,23 @@
+#home,#contactus{
+	display: inline;
+}
+#home,#contactus:hover{
+cursor: pointer;
+}
+#name-warning,#email-warning,#organization-warning,#phone-warning,#text-warning{
```

Comment:
Their can be a single class for error, which can be assigned to all respective divs